scripts/skills/racial/rf_zombie_racial.nut

```diff
@@ -97,4 +97,9 @@ this.rf_zombie_racial <- ::inherit("scripts/skills/skill", {
 		baseProperties.FatigueEffectMult = this.m.FatigueEffectMult;
 		baseProperties.FatigueDealtPerHitMult += this.m.FatigueDealtPerHitMultModifier;	// In vanilla normal zombies don't have this bonus
 	}
+
+	function onUpdate( _properties )
```

Comment:
I think a cleaner solution is to remove Encumbrance from Undead units.

Reply:
For this we can change how we add Encumbrance in `actor.onInit` and only add it if the entity isn't kind of `zombie` or `skeleton`. Alternatively, inside Encumbrance effect, we can make it not apply if `FatigueEffectMult == 0`.

Reply:
Okay, while I still see this bug returning even without encumbrance, im ok waiting until it happens.